modules/administration_manual/pages/configuration/server/occ_app_command.adoc

```diff
@@ -238,6 +239,65 @@ Time how long the ban will be active if triggered.
 | Default | 300
 |===
 
+[[data_exporter]]
+== Data Exporter
+
+Marketplace URL: https://marketplace.owncloud.com/apps/data_exporter[Data Exporter]
```

Comment:
This will not be released to the marketplace - we better remove this link and replace with a link to the github release page